pkg/exec/schema/exec.json

```diff
@@ -72,6 +72,13 @@
             "type": "string"
           }
         },
+        "envs": {
```

Comment:
A million brownie points for remembering to update the schema that VS Code uses to do autocomplete 💯